source/index.html

```diff
@@ -1,11 +1,182 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="main header">
+    <a class="main-header__logo">
+      <img src="img/logo-mobile.svg"
+  width="33.32" height="37.61">
+      <img src="img/footer-logo 2.svg"
+  width="101 "height="18">
+    </a>
+    <nav class="main nav">
+      <button
+  class="main-nav__toggle"
+  type="button"><span
+  class="visually-hidden">Открыть
+  меню</span></button>
+        <div
+  class="main-nav__wrapper">
+          <ul class="main-nav__list
+  site-list">
+            <li
+  class="site-list__item
+  site-list__item--active">
+              <a>
+                ГЛАВНАЯ
+              </a>
+            </li>
+            <li
+  class="site-list__item">
+            <a href="catalog.html">
+              КАТАЛОГ ПРОДУКЦИИ
+            </a>
+          </li>
+          <li
+  class="site-list__item">
+              <a href="programa.html">
+                ПОДБОР ПРОГРАММЫ
+              </a>
+            </li>
+          </ul>
+          <div class="main-nav__list user-list">
 
+          </div>
+        </div>
+      </nav>
+    </header>
+      <p class="stats__slogan">
+        <h1>Функциональное<br>
+        питание для котов</h1>
+      </p>
```

Comment:
Заголовок H не может быть внутри p и наоборот. Это сочетание не корректно. Можно сделать вот так
```suggestion
     <h1 class="stats__slogan">Функциональное<br>
        питание для котов</h1>
```